src/Raven.Server/Documents/Archive/ArchivalStorage.cs

```diff
@@ -0,0 +1,341 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Raven.Client;
+using Raven.Client.Documents.Attachments;
+using Raven.Client.Documents.Operations.Counters;
+using Raven.Server.Documents.Replication.ReplicationItems;
+using Raven.Server.Documents.TimeSeries;
+using Raven.Server.ServerWide.Context;
+using Sparrow;
+using Sparrow.Json;
+using Sparrow.Logging;
+using Voron;
+using Sparrow.Json.Parsing;
+using Voron.Impl;
+using Bits = Sparrow.Binary.Bits;
+
+namespace Raven.Server.Documents.Archival
+{
+    public unsafe class ArchivalStorage
+    {
+        private const string DocumentsByArchive = "DocumentsByArchive";
```

Comment:
`DocumentsByArchive` is a bit meaningless. Maybe `DocumentsByArchiveDate` or `DocumentsByArchiveDateTime`?

Reply:
https://github.com/ravendb/ravendb/pull/15715/commits/27c40d19ab63ef2794ecc0e2d3f938a8a4e32247
Fixed